### PR TITLE
Add com.katawa_shoujo.KatawaShoujo

### PR DESCRIPTION
**Pull request by FakeShemp**
My first attempt. Probably not the most elegant manifest, I'm open to any suggestions that make it better. 

Help with solving the issue that both i386 and x86_64 versions of the game are bundled in the same archive in a more simple way would be helpful.

I also couldn't get it to start with hardware accelerated graphics. I would appreciate help there too.

### CODE REVIEW

File: com.katawa_shoujo.KatawaShoujo.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+			The story is told through the perspective of the main character, using a first person narrative. The game uses a traditional text and sprite-based visual novel model with an ADV text box. 
+		</p>
+		<p>
+			Katawa Shoujo contains adult material, and was created using the Ren'Py scripting system. It is the product of an international team of amateur developers, and is available free of charge under the Creative Commons BY-NC-ND License.
+		</p>
+	</description>
+
+	<launchable type="desktop-id">com.katawa_shoujo.KatawaShoujo.desktop</launchable>
+
+	<categories>
```
Comment (barthalion) [inline com.katawa_shoujo.KatawaShoujo.appdata.xml]: No need to define it here if they are also in desktop file AFAIK.

**Comment by FakeShemp** [comment]
@barthalion I don't mind at all, that'd be awesome.

**Comment by FakeShemp** [comment]
@bilelmoussaoui Great suggestions. Made them :)

**Comment by nedrichards** [comment]
bot, build com.katawa_shoujo.KatawaShoujo

File: com.katawa_shoujo.KatawaShoujo.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+	<description>
+		<p>
+			Katawa Shoujo is a bishoujo-style visual novel set in the fictional Yamaku High School for disabled children, located somewhere in modern Japan. Hisao Nakai, a normal boy living a normal life, has his life turned upside down when a congenital heart defect forces him to move to a new school after a long hospitalization. Despite his difficulties, Hisao is able to find friends—and perhaps love, if he plays his cards right. There are five main paths corresponding to the 5 main female characters, each path following the storyline pertaining to that character.
+		</p>
+		<p>
+			The story is told through the perspective of the main character, using a first person narrative. The game uses a traditional text and sprite-based visual novel model with an ADV text box. 
+		</p>
+		<p>
+			Katawa Shoujo contains adult material, and was created using the Ren'Py scripting system. It is the product of an international team of amateur developers, and is available free of charge under the Creative Commons BY-NC-ND License.
```
Comment (nedrichards) [inline com.katawa_shoujo.KatawaShoujo.appdata.xml]: isn't this BY-NC-SA? Based on the licence code above.

Comment (FakeShemp) [reply com.katawa_shoujo.KatawaShoujo.appdata.xml]: It probably is, yes :)

**Comment by nedrichards** [comment]
bot, build com.katawa_shoujo.KatawaShoujo

**Comment by nedrichards** [comment]
Other than the description vs project_licence thing this lgtm.

@barthalion did you want to hold on the python thing? or push a PR afterwards?

File: katawa-shoujo-launcher-i386.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec "/app/lib/linux-i686/python" "-EO" "/app/Katawa Shoujo.py" "$@"
```
Comment (TingPing) [inline katawa-shoujo-launcher-i386.sh]: Might as well de-bundle python assuming the runtime version works.

**Comment by barthalion** [comment]
Not ready yet:
```
Traceback (most recent call last):
  File "/app/share/renpy/renpy.py", line 199, in <module>
    main()
  File "/app/share/renpy/renpy.py", line 196, in main
    renpy.bootstrap.bootstrap(renpy_base)
  File "/app/share/renpy/renpy/bootstrap.py", line 281, in bootstrap
    renpy.import_all()
  File "/app/share/renpy/renpy/__init__.py", line 419, in import_all
    import renpy.text.text
  File "/app/share/renpy/renpy/text/text.py", line 32, in <module>
    from _renpybidi import log2vis, WRTL, RTL, ON  # @UnresolvedImport
ImportError: No module named _renpybidi
```

Mostly likely a packaging mistake on my side.

**Comment by barthalion** [comment]
bot, build com.katawa_shoujo.KatawaShoujo

**Comment by barthalion** [comment]
bot, build com.katawa_shoujo.KatawaShoujo

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/com.katawa_shoujo.KatawaShoujo

Thanks!

**Comment by FakeShemp** [comment]
No, thank you <3

**Pull request closed without merging**